apps/src/tutorialExplorer/util.jsx

```diff
@@ -5,6 +5,9 @@ import * as utils from '../utils';
 // Sorting for tutorials.
 export const TutorialsSortBy = utils.makeEnum('default', 'popularityrank', 'displayweight');
 
+// Orgname value.
+export const TutorialsOrgName = utils.makeEnum('all');
```

Comment:
Why do we have an enum with a single value? Presumably we're planning on adding more orgs?

Reply:
Not sure yet, but I did it mainly so that it'd be consistent with how `TutorialsSortBy` is done, above, since it works very similarly.